openedx/features/edly/admin.py

```diff
@@ -0,0 +1,41 @@
+"""
+Django Admin pages for Edly.
+"""
+
+from django.contrib import admin
+
+from openedx.features.edly.models import EdlyOrganization, EdlySubOrganization
+
+
+class EdlyOrganizationAdmin(admin.ModelAdmin):
+    list_display = ['name', 'slug', 'created', 'modified']
+
+
+admin.site.register(EdlyOrganization, EdlyOrganizationAdmin)
+
+
+class EdlySubOrganizationAdmin(admin.ModelAdmin):
+    search_fields = ['lms_site__domain', 'lms_site__name', 'studio_site__domain', 'studio_site__name']
+    list_display = [
+        'edly_organization_name',
+        'edly_organization_slug',
+        'edx_organization_name',
+        'edx_organization_short_name',
+        'created',
+        'modified',
+    ]
+
+    def edly_organization_name(self, obj):
```

Comment:
how about using `edly_organization__name` instead of this method. Same for other fields

Reply:
That's not working. giving the following error:
`<class 'openedx.features.edly.admin.EdlySubOrganizationAdmin'>: (admin.E108) The value of 'list_display[1]' refers to 'edly_organization__name', which is not a callable, an attribute of 'EdlySubOrganizationAdmin', or an attribute or method on 'edly.EdlySubOrganization'.`

Reply:
Ok.